druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/entity/CacheActionRunnerFlatBuffer.java

```diff
@@ -0,0 +1,124 @@
+package com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.entity;
+
+import com.google.common.base.Strings;
+import com.google.flatbuffers.Table;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import com.yahoo.maha.maha_druid_lookups.query.lookup.DecodeConfig;
+import com.yahoo.maha.maha_druid_lookups.query.lookup.namespace.RocksDBExtractionNamespace;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.LookupService;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.MonitoringConstants;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.RocksDBManager;
+import com.metamx.common.logger.Logger;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.schema.BaseSchemaFactory;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.schema.flatbuffer.FlatBufferSchemaFactory;
+import com.yahoo.maha.maha_druid_lookups.server.lookup.namespace.schema.flatbuffer.FlatBufferWrapper;
+import org.rocksdb.RocksDB;
+
+import java.util.Optional;
+
+public class CacheActionRunnerFlatBuffer implements BaseCacheActionRunnerChain {
+
+    private static final Logger LOG = new Logger(CacheActionRunner.class);
```

Comment:
Let the logger be CacheActionRunnerFlatBuffer.class

Reply:
fixed